README.md

```diff
@@ -146,42 +146,31 @@ If it's not `healthy` you should test your logstash configuration.
-
-There are two different types of configurations that can be supplied to logstash-mesos.
-They are both based on the [same configuration language](https://www.elastic.co/guide/en/logstash/current/configuration.html) as logstash,
- but adds some custom behaviour because of the fact that it is running inside a mesos executor.
- 
-The [scheduler's web UI](#gui) will provide a helpful interface for configuring logstash-mesos the way you want.
-
-### Slave Configuration
-Slave (or "Host") configurations will be propagated to all running logstash-instances. They are a good place to put output-blocks and to specify slave log-files to be monitored.
-You can use the special key 'host-path' to configure file-plugins inside logstash input-blocks to indicate that a file should be logged from the slave itself (instead of the executor's docker container).
-Example:
-```
-input { file { "host-path" => "/var/log/hello.log" } }
-```
-This will configure logstash to monitor the file `/var/log/hello.log` on all slaves. Note: this requires that the logstash framework is configured to use `/var/log` as a volume.
-
-### Docker Configuration
-Docker configurations will be propagated to slaves running docker containers with image names matching the docker configuration. Note that tags are usually included in docker image names.
-
-Example: a slave running a docker container with the nginx:latest docker image will be provided a docker configuration with 'applicable image name' set to nginx:latest, if it exists.
-
-You can use the special key 'docker-path' to configure file-plugins inside logstash input-blocks to indicate that a file should be logged from within the docker container itself.
-
-Example:
-```
-input { file { "docker-path" => "/var/log/hello.log" } }
-```
-
-This will configure logstash to monitor the file `/var/log/hello.log` inside all applicable docker containers.
+### Configuration
+
+The Logstash framework is configured at the time that the scheduler is started. Each configuration option can be passed in a large number of ways:
+
+1. Command-line arguments, e.g. `java -jar logstash-mesos-scheduler.jar --logstash.heapSize=64`
+2. Environment variables, e.g. `LOGSTASH_HEAP_SIZE=64 java -jar logstash-mesos-scheduler.jar`
+3. A properties file, e.g. `echo 'logstash.heap-size=64' > ./application.properties && java -jar logstash-mesos-scheduler.jar` 
+4. ...
+
+| Command-line argument            | Environment variable           | What it does                                                                                                               |
+| -------------------------------- | ------------------------------ | -------------------------------------------------------------------------------------------------------------------------- |
+| `--zk-url U`                     | `ZK_URL=U`                     | The Logstash framework will find Mesos using ZooKeeper at URL `U`, which must be in the format `zk://host:port/zkNode,...` |
```

Comment:
https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html
> When running in a new environment, an application.properties can be provided outside of your jar that overrides the `name`; and for one-off testing, you can launch with a specific command line switch (e.g. `java -jar app.jar --name="Spring"`).

Seems pretty clear to me